src/contract.rs

```diff
@@ -229,6 +284,7 @@ fn create_ask(
 
     if ask_order.base.denom.ne(&contract_info.base_denom) {
         ask_order.class = AskOrderClass::Convertible {
+            converted_base: None,
             status: AskOrderStatus::PendingIssuerApproval,
         };
     };
```

Comment:
can we add another attribute here for saying this ask is using a convertible denom? or how do we know?

Reply:
Absolutely. I can add the `class` to the attributes to the response when creating an ask. it's kinda there already since the `base` attribute would be a convertible denom instead of the target base denom. but I can add `class` so that it is very explicit if that works?